build/ps-entrypoint.sh

```diff
@@ -226,7 +227,14 @@ if [ "$1" = 'mysqld' -a -z "$wantHelp" ]; then
 		echo 'Initializing database'
 		# we initialize database into $TMPDIR because "--initialize-insecure" option does not work if directory is not empty
 		# in some cases storage driver creates unremovable artifacts (see K8SPXC-286), so $DATADIR cleanup is not possible
-		"$@" --initialize-insecure --skip-ssl --datadir="$TMPDIR"
+		init_args=(--initialize-insecure --datadir="$TMPDIR")
+		# --skip-ssl was removed in 8.4
+		if [ "$MYSQL_MINOR_VERSION" -lt 4 ]; then
```

Comment:
@egegunes @pooknull, maybe we can remove it for all versions ...

Reply:
I agree